src/lib.rs

```diff
@@ -426,7 +426,7 @@ pub enum Interrupt {
     #[doc = "91 - PORTE"]
     PORTE,
 }
-unsafe impl ::bare_metal::Nr for Interrupt {
+unsafe impl ::cortex_m::interrupt::Nr for Interrupt {
```

Comment:
This change based off this comment: https://github.com/rust-embedded/bare-metal/issues/22#issuecomment-617349447